openedx/core/djangoapps/user_api/accounts/tests/test_retirement_views.py

```diff
@@ -560,6 +560,68 @@ def test_does_not_delete_users_in_process(self):
         self.assert_status_and_count(statuses, len(statuses))
 
 
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Account APIs are only supported in LMS')
+class TestPartnerReportingPut(RetirementTestCase, ModuleStoreTestCase):
+    """
+    Tests the partner reporting list endpoint
+    """
+
+    def setUp(self):
+        super(TestPartnerReportingPut, self).setUp()
+        self.test_superuser = SuperuserFactory()
+        self.course = CourseFactory()
+        self.course_awesome_org = CourseFactory(org='awesome_org')
+        self.courses = (self.course, self.course_awesome_org)
+        self.headers = build_jwt_headers(self.test_superuser)
+        self.url = reverse('accounts_retirement_partner_report')
+        self.headers['content_type'] = "application/json"
+        self.maxDiff = None
+        self.pending = RetirementState.objects.get(state_name='RETIRING_EMAIL_LISTS')
```

Comment:
It doesn't seem like "self.pending" is the correct variable name here, but also RETIRING_EMAIL_LISTS is less than ideal since I assume the correct state for calling this PUT API will sound more like "POPULATING_PARTNER_QUEUE" or "RETIRING_POPULATING_PARTNER_QUEUE".